analyses/gene_match/qc_ensg_hugo_pmtl_mapping.Rmd

```diff
@@ -14,7 +14,7 @@ suppressPackageStartupMessages({
 ## Read ensg-hugo-pmtl-mapping.tsv and OpenPedCan SNV, CNV, TPM, and fusion data
 
 ```{r read_open_ped_can_data}
-open_ped_can_data_path <- '~/OpenPedCan-analysis/data/'
+open_ped_can_data_path <- '../../data/'
```

Comment:
@sangeetashukla, trailing backslash is unnecessary. R will add when joining to create a path. Will still work but you will have double backslashes as shown in the previous error message that I posted.
 set to this: `open_ped_can_data_path <- '../../data'`